examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1185,7 +1185,6 @@ def main(args):
     # now we will add new LoRA weights to the attention layers
     unet_lora_config = LoraConfig(
         r=args.rank,
-        use_dora=args.use_dora,
```

Comment:
Can you explain this change?

Reply:
## I was getting error because older versions of PEFT library(<0.9.0) don't support the [use_dora] parameter in LoraConfig 
## This code can fix this ,so now the code will work regardless of PEFT version, with use_dora enabled only if the PEFT version is 0.9.0 or higher.

`def get_lora_config(rank, use_dora, target_modules):
    base_config = {
        "r": rank,
        "lora_alpha": rank,
        "init_lora_weights": "gaussian",
        "target_modules": target_modules,
    }
    
    if use_dora:
        try:
            from packaging import version
            import peft
            if version.parse(peft.__version__) >= version.parse("0.9.0"):
                base_config["use_dora"] = True
            else:
                print("Warning: DoRA requires PEFT >= 0.9.0. Continuing without DoRA.")
        except ImportError:
            print("Warning: Could not verify PEFT version. Continuing without DoRA.")
            
    return LoraConfig(**base_config)`

Reply:
Then you should upgrade your `peft` installation.